lib/db/leveldb_dbinstance.go

```diff
@@ -923,6 +903,19 @@ func unmarshalTrunc(bs []byte, truncate bool) (FileIntf, error) {
 	return tf, err
 }
 
+func unmarshalVersionList(data []byte) (VersionList, bool) {
+	var vl VersionList
+	if err := vl.Unmarshal(data); err != nil {
+		l.Debugln("unmarshal error:", err)
+		return VersionList{}, false
+	}
+	if len(vl.Versions) == 0 {
+		l.Debugln("empty version list")
```

Comment:
I can't see how these debug logs are useful, they would appear thousands and thousands of times in a db walk, amd include no indication which file they are talking about etc. I'd just remove them.

Reply:
It's the same territory as the error above, it's useless-ish but it's never supposed to happen

Reply:
Then change it to a panic?